app/stores/customer-rare.ts

```
@@ -247,221 +243,229 @@ interface ISavedMealRatingResult {
+								beverageTag: null,
+								recipeTag: null,
+							};
+							meal.popular = {
+								isNegative: false,
+								tag: null,
+							};
+						}
 					}
+					// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+					oldState.persistence.meals = oldState.page.selected;
+					delete oldState.persistence.selected;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)